vignettes/exporting_options.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "Exporting Options"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Exporting Options}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r, include = FALSE}
+library(httptest)
+# Make sure REDCapTidieR is loaded so httptest can find redact.R
+library(REDCapTidieR)
+
+# Use fake credentials if mocks exist, otherwise use real credentials to create mocks
+fake <- dir.exists("exporting_options")
+
+creds <- REDCapTidieR:::get_credentials(
+  c("REDCAP_URI", "SUPERHEROES_REDCAP_API", "REDCAPTIDIER_CLASSIC_API"),
+  fake = fake
+)
+
+redcap_uri <- creds$REDCAP_URI
+superheroes_token <- creds$SUPERHEROES_REDCAP_API
+classic_token <- creds$REDCAPTIDIER_CLASSIC_API
+
+start_vignette("exporting_options")
+```
+
+## Exporting REDCapTidieR Outputs
+
+In this section, we will be discussing options available to export your data once you've successfully created a [supertibble](glossary.html#supertibble). For more information on how to make a [supertibble](glossary.html#supertibble), refer to the [Getting Started](REDCapTidieR.html) vignette.
+
+Have an idea or suggestion for other formats or methods you wish to see? Please submit a [feature request](https://github.com/CHOP-CGTInformatics/REDCapTidieR/issues/new/choose) to us on GitHub!
+
+### Exporting to XLSX
+
+A common need for analysts working with REDCap data is the ability to sharing share it in an accessible and interpretable format. Excel spreadsheets are one such solution, and REDCapTidieR supports this using the `write_redcap_xlsx()` function. `write_redcap_xlsx()` is built by leveraging the powerful [`openxlsx2`](https://janmarvin.github.io/openxlsx2/) package. Let's explore how to leverage this function and what to expect from an output by starting with the superheroes dataset:
+
+``` r
+superheroes_token <- "123456789ABCDEF123456789ABCDEF04"
+redcap_uri <- "https://my.institution.edu/redcap/api/"
+```
+
+```{r}
+supertbl <- read_redcap(redcap_uri, superheroes_token)
+
+supertbl %>%
+  rmarkdown::paged_table()
+```
+
+The goal of `write_redcap_xlsx()` is to provide a single .xlsx file where each [data tibble](glossary.html#data-tibble) (i.e., what is listed under the `redcap_data` column) is displayed in its own sheet. Let's pass the `supertbl` defined above through the function and investigate the default output:
```

Comment:
I don't know if I like my language that much better but something about "passing through the function" just seems off to me
```suggestion
The goal of `write_redcap_xlsx()` is to provide a single `.xlsx` file where each [data tibble](glossary.html#data-tibble) (i.e., what is listed under the `redcap_data` column) is displayed in its own sheet. Let's try it with the superheroes `supertbl` created above:
```

Reply:
That's fine with me 👍